src/class/hid/hid.h

```diff
@@ -350,6 +350,22 @@ typedef enum
 
 /// @}
 
+//--------------------------------------------------------------------+
+// NKROKeyboard
+//--------------------------------------------------------------------+
+/** \addtogroup ClassDriver_HID_NKROKeyboard NKROKeyboard
+ *  @{ */
+
+/// HID NKROKeyboard Report
+typedef struct TU_ATTR_PACKED

```

Comment:
Could I ask why you decide to choose this layout with 13 key array and 1 custom key ?

Reply:
thank you, i've change it to 256key layout (32bytes)